lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -89,9 +89,10 @@ class CategoryRenderer {
     // Add list of warnings or singular warning
     const warningsEl = this.dom.createChildOf(titleEl, 'div', 'lh-warnings');
     if (warnings.length === 1) {
-      warningsEl.textContent = `Warning: ${warnings.join('')}`;
+      // TODO(i18n): Maybe need to construct this differently
```

Comment:
> TODO(i18n): Maybe need to construct this differently

if warnings are full sentences this should be fine

Reply:
alright, removed